app/views/active_admin/_site_header.html.erb

```diff
@@ -21,7 +21,11 @@
   <div id="user-menu" class="z-50 hidden min-w-max bg-white rounded shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none dark:bg-gray-700 py-1 text-sm text-gray-700 dark:text-gray-200" aria-labelledby="user-menu-button">
     <ul>
       <% if current_active_admin_user? %>
-        <li><%= auto_link current_active_admin_user, class: "block px-2.5 py-2 no-underline text-gray-700 hover:bg-gray-100 hover:text-gray-900 dark:text-white dark:hover:bg-gray-600 dark:hover:text-white" %></li>
+        <% if auto_url_for(current_active_admin_user) %>
```

Comment:
@javierjulio @mgrunberg do you have any idea why this might randomly fail with version 7.1?

The `auto_url_for(resource)` function is called at the very start of `auto_link`, which was previously used without issues. I suspect the issue might relate to autoloading. The only difference could be the invocation of `display_name`, which is used in the default parameters of `auto_link`.

Reply:
@tagliala sorry, I do not. I don't think it's necessary that we do this change. The template is not meant to account for every scenario. We have a good default with the admin user link. We don't need anything else out of the box because the rest is meant for a user to update to meet their needs that's why we provide the template.